### PR TITLE
Handle new order statuses - returned & partially returned

### PR DESCRIPTION
**Pull request by mmarkusik**
I want to merge this change because it adds displaying of new order statuses - returned & partially returned

**PR intended to be tested with API branch:** 

### Pull Request Checklist

1. [ ] All visible strings are translated with proper context.
1. [ ] All data-formatting is locale-aware (dates, numbers, and so on).
1. [ ] Translated strings are extracted.
1. [ ] Number of API calls is optimized.
1. [ ] The changes are tested.
1. [ ] Data-test are added for new elements.
1. [ ] Type definitions are up to date.
1. [ ] Changes are mentioned in the changelog.

### Test environment config

API_URI=https://master.staging.saleor.rocks/graphql/

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-1860 Add new statuses for order](https://app.clickup.com/t/2549495/SALEOR-1860)

File: src/misc.ts [hunk 1]
```diff
@@ -189,6 +197,16 @@ export const transformOrderStatus = (
...
       };
+    case OrderStatus.PARTIALLY_RETURNED:
+      return {
+        localized: intl.formatMessage(orderStatusMessages.partiallyReturned),
+        status: StatusType.NEUTRAL
+      };
+    case OrderStatus.RETURNED:
+      return {
+        localized: intl.formatMessage(orderStatusMessages.returned),
+        status: StatusType.ERROR
```
Comment (jwm0) [inline src/misc.ts:208]: Should the status be `ERROR` for a RETURNED order status?

**Comment by mmarkusik** [comment]
Still need a verification from @pwgryglak when he's back from holiday 🌴

File: src/misc.ts (hunk 1 above)
Comment (dominik-zeglen) [reply src/misc.ts:208]: Can't find it on the designs. 
@pwgryglak which dot color should we display on the order list if order is refunded / partially refunded?

Comment (jwm0) [reply src/misc.ts:208]: I talked with @pwgryglak and he mentioned the tag should be informational, so I believe `StatusType.NEUTRAL` should do the trick.

**Review by dominik-zeglen — APPROVED** [review]

**Review by krzysztofwolski — APPROVED** [review]

**Pull request merged**